packages/core/src/Footer/Logo.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import PropTypes from "prop-types";
+import clsx from "clsx";
+
+import A from "../A";
+import { Grid, Typography } from "@material-ui/core";
+
+import useStyles from "./useStyles";
+
+function Logo({ variant, organizationLogo, ...props }) {
+  const classes = useStyles(props);
+
+  return (
+    <Grid container alignItems="baseline" className={classes.organization}>
+      <Grid item>
+        <Typography variant="h1">
+          <A href={organizationLogo.url}>
+            <img
+              src={organizationLogo.image.url}
+              alt={organizationLogo.image.alt}
+              className={classes.organizationLogo}
+            />
+          </A>
+        </Typography>
+      </Grid>
+      {variant === "divider" && (
```

Comment:
Not sure about using `variant` for this. Why not just a boolean prop e.g. `divider` defaulting to `true`